packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputPaymentMethod.tsx

```diff
@@ -42,41 +36,32 @@ export const CoinmarketFormInputPaymentMethod = ({ label }: CoinmarketFormInputD
     };
 
     return (
-        <>
-            <CoinmarketFormInputLabel label={label} />
-            <CoinmarketFormInputInner>
-                <Controller
-                    name={FORM_PAYMENT_METHOD_SELECT}
-                    defaultValue={defaultPaymentMethod}
-                    control={control as Control<CoinmarketBuySellFormProps>}
-                    render={({ field: { onChange, value } }) => (
-                        <Select
-                            value={value}
-                            onChange={onChange}
-                            options={paymentMethods}
-                            formatOptionLabel={(option: CoinmarketPaymentMethodListProps) => (
-                                <CoinmarketFormOption>
-                                    <CoinmarketFormOptionLabel>
-                                        {option.value !== '' && quotes && quotes.length > 0 ? (
-                                            <CoinmarketPaymentPlainType
-                                                method={option.value}
-                                                methodName={option.label}
-                                            />
-                                        ) : (
-                                            getEmptyMethodsLabel()
-                                        )}
-                                    </CoinmarketFormOptionLabel>
-                                </CoinmarketFormOption>
-                            )}
-                            data-testid="@coinmarket/form/payment-method-select"
-                            isClearable={false}
-                            isSearchable
-                            isDisabled={isFormInvalid}
-                        />
-                    )}
+        <Controller
+            name={FORM_PAYMENT_METHOD_SELECT}
+            defaultValue={defaultPaymentMethod}
+            control={control as Control<CoinmarketBuySellFormProps>}
+            render={({ field: { onChange, value } }) => (
+                <Select
+                    value={value}
+                    onChange={onChange}
+                    options={paymentMethods}
+                    labelLeft={<Translation id={label as TranslationKey} />}
```

Comment:
A label can be also `undefined` and would be also nice without type forcing...
![obrazek](https://github.com/user-attachments/assets/20cfe917-ac63-4a62-b9db-0ca69c14f9b7)